.github/workflows/release.yaml

```
@@ -25,123 +25,129 @@ jobs:
+            #       cache-dependency-path: ./yarn.lock
+            #       registry-url: https://registry.npmjs.org
+
+            # - name: Publish to npm
+            #   run: |
+            #       yarn publish --no-git-tag-version --tag latest
+            #   env:
+            #       NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
             - name: Bump Version
               id: bump
               run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:40: Double quote to prevent globbing and word splitting

(shellcheck)